src/Components/Blueprints/BlueprintActionsMenu.tsx

```diff
@@ -60,11 +71,36 @@ export const BlueprintActionsMenu: React.FunctionComponent<
         >
           Edit details
         </DropdownItem>
-        <DropdownItem>Download blueprint (.json)</DropdownItem>
+        <DropdownItem
+          onClick={() => {
+            trigger({ id: selectedBlueprintId || '' });
+          }}
+        >
+          Download blueprint (.json) <BetaLabel />
```

Comment:
Since we've got Unleash in place, do we really need the `<BetaLabel />`? I think we can just ship this straight to prod-stable when we're ready.

Reply:
I think that's the thing ... we should ship it to prod-stable with beta label :P
It should mark the exported schema as "we can't promise it'll be importable next month".